tests/integration/mlflow-setup.py

```diff
@@ -0,0 +1,16 @@
+from random import randint, random
+
+import mlflow
+from mlflow import log_metric, log_param
+
+mlflow.set_tracking_uri("http://mlflow-server:5000")
```

Comment:
this tracking URI should be the database URI instead, and then you can run the script directly, with no intermediate MLflow server